### PR TITLE
updated the way the chat listing button is placed

### PR DESCRIPTION
**Pull request by DShivansh**
Since the button used to be hidden behind the iphoneX's status bar so i now programmatically calculate the height of the of the status bar and place the button accordingly.

Changes: [i changed the static height at which the skill listing button is placed and made it dynamic]

Screenshots for the change:

### CODE REVIEW

**Comment by chashmeetsingh** [comment]
Can you please add a screenshot for the change, before and after the change? @DShivansh

**Comment by DShivansh** [comment]
screen shot 2018-01-08 at 12 37 45 pm
screen shot 2018-01-08 at 12 47 44 pm
these are the screen shots

File: Susi/Controllers/ChatViewController/ChatVCMethods.swift [hunk 1]
```diff
@@ -158,7 +158,8 @@ extension ChatViewController {
     func addSkillListingButton() {
         view.addSubview(susiSkillListingButton)
         view.addConstraintsWithFormat(format: "H:[v0(36)]-8-|", views: susiSkillListingButton)
-        view.addConstraintsWithFormat(format: "V:|-28-[v0(36)]", views: susiSkillListingButton)
+        let height = UIApplication.shared.statusBarFrame.size.height
+        view.addConstraintsWithFormat(format: "V:|-\(Int(height)+1)-[v0(36)]", views: susiSkillListingButton)
```
Comment (chashmeetsingh) [inline Susi/Controllers/ChatViewController/ChatVCMethods.swift]: Add 4 instead of 1. 
Rest looks good.

Comment (DShivansh) [reply Susi/Controllers/ChatViewController/ChatVCMethods.swift]: cool

**Pull request merged**